packages/angular/ssr/src/app.ts

```diff
@@ -6,25 +6,34 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
-import { StaticProvider, ɵConsole, ɵresetCompiledComponents } from '@angular/core';
-import { ɵSERVER_CONTEXT as SERVER_CONTEXT } from '@angular/platform-server';
+import { StaticProvider, ɵresetCompiledComponents } from '@angular/core';
 import { ServerAssets } from './assets';
-import { Console } from './console';
 import { Hooks } from './hooks';
 import { getAngularAppManifest } from './manifest';
+import { RenderMode } from './routes/route-config';
 import { ServerRouter } from './routes/router';
 import { REQUEST, REQUEST_CONTEXT, RESPONSE_INIT } from './tokens';
 import { InlineCriticalCssProcessor } from './utils/inline-critical-css';
 import { AngularBootstrap, renderAngular } from './utils/ng';
 
 /**
- * Enum representing the different contexts in which server rendering can occur.
+ * A mapping of `RenderMode` enum values to corresponding string representations.
+ *
+ * This record is used to map each `RenderMode` to a specific string value that represents
+ * the server context. The string values are used internally to differentiate
+ * between various rendering strategies when processing routes.
+ *
+ * - `RenderMode.Prerender` maps to `'ssg'` (Static Site Generation).
+ * - `RenderMode.Server` maps to `'ssr'` (Server-Side Rendering).
+ * - `RenderMode.AppShell` maps to `'app-shell'` (pre-rendered application shell).
+ * - `RenderMode.Client` maps to an empty string `''` (Client-Side Rendering, no server context needed).
  */
-export enum ServerRenderContext {
-  SSR = 'ssr',
-  SSG = 'ssg',
-  AppShell = 'app-shell',
-}
+const SERVER_CONTEXT_VALUE: Record<RenderMode, string> = {
+  [RenderMode.Prerender]: 'ssg',
+  [RenderMode.Server]: 'ssr',
```

Comment:
Consider: If we've renamed `RenderMode` to use `Prerender` and `Server`, should we update these string values as well?

Reply:
No, as these are the values that Aurora track.